registrations/utils.py

```diff
@@ -329,3 +329,41 @@ def create_web_store_product_accounting(product_id: str, product_accounting_data
         raise WebStoreAPIError(api_error_message)
 
     return resp_json
+
+
+def get_web_store_order(order_id: str):
+    order_api_client = WebStoreOrderAPIClient()
+
+    try:
+        resp_json = order_api_client.get_order(order_id=order_id)
+    except RequestException as request_exc:
+        api_error_message = get_web_store_api_error_message(request_exc.response)
+        raise WebStoreAPIError(
+            api_error_message, getattr(request_exc.response, "status_code", None)
+        )
```

Comment:
Since this is directly caused by `RequestException`, you should use `raise ... from request_exc`. Same goes for `get_web_store_payment`.

Reply:
Changed as suggested.